terraform-environments/instance.tf

```diff
@@ -2,7 +2,9 @@ resource "aws_security_group" "security" {
   name        = "ec2_security2"
```

Comment:
This file should prob be renamed. instance could be anything

Reply:
its a lot more used by tani and ari - i just created the outline for it, and i believe they renamed it in their branch